src/middle/Stan_math_signatures.ml

```diff
@@ -1592,7 +1592,8 @@ let () =
   add_nullary "log10" ;
   add_unqualified ("log10", ReturnType UComplex, [UComplex], AoS) ;
   add_nullary "log2" ;
-  add_unqualified ("log_determinant", ReturnType UReal, [UMatrix], SoA) ;
+  add_unqualified ("log_determinant", ReturnType UReal, [UMatrix], AoS) ;
```

Comment:
Why did `log_determinant` change from SoA to AoS?

Reply:
I didn't mean to touch that and don't recall doing anything to it. I can revert that. What do those mean?

Reply:
They indicate whether or not the function arguments can be Struct of Arrays matrices when the compiler is performing optimizations. If the new `log_determinant_spd` has the correct templates to be used with a `var_matrix` it can also be marked as SoA